app/controllers/content_items_controller.rb

```diff
@@ -70,6 +70,6 @@ def return_arbiter_error(status, exception)
     else
       item.errors.add("url_arbiter_registration", "#{exception.response.code}: #{exception.response.raw_body}")
     end
-    render :json => PrivateContentItemPresenter.new(item), :status => status
+    render :json => { errors: item.errors.as_json }, :status => status
```

Comment:
Now that we're no longer returning the rest of the response JSON, there's no need to construct an unsaved ContentItem.  It would be simpler to just construct a hash with the error details in it.